oauth2client/flask_util.py

```diff
@@ -191,6 +192,21 @@ def requires_calendar():
 __author__ = 'jonwayne@google.com (Jon Wayne Parrott)'
 
 DEFAULT_SCOPES = ('email',)
+CREDENTIALS_KEY = 'google_oauth2_credentials'
```

Comment:
Users could modify them if needed to avoid conflicts, but I imagine that's a pretty slim situation because they're prefixed with `google_`. What do you think?